model/src/main/proto/javaclasses/exlibris/c/events.proto

```diff
@@ -36,12 +36,12 @@ import "google/protobuf/timestamp.proto";
 
 // An event fired when librarian adds new book to the library.
 //
-// React on BookAdded
-//
 message InventoryCreated {
 
     // The same as the bookId.
```

Comment:
Change that doc. There just ID of inventory. Another information must be in `InventoryId`.